css/src/mixins/media-queries.scss

```diff
@@ -40,3 +40,10 @@
 		@content;
 	}
 }
+
+// Please avoid using this mixin if possible. It doesn't follow our the mobile-first approach to writing CSS.
```

Comment:
nit: `the` should be removed.

Reply:
Actually decided to remove the `our` instead of `the` from the sentence: "It doesn't follow **our** the mobile-first approach to writing CSS."